internal/lib/lib.go

```diff
@@ -620,8 +651,8 @@ func VSVipChecksum(FQDNs []string, IPAddress string, networkName string) uint32
 	if IPAddress != "" {
 		checksum += utils.Hash(IPAddress)
 	}
-	if networkName != "" {
-		checksum += utils.Hash(networkName)
+	if len(networkNames) != 0 {
+		checksum = utils.Hash(utils.Stringify(networkNames))
```

Comment:
@sudswasavi i believe this check takes care of the case when the vipNetworkList is empty.

Reply:
Does this mean that if fqdn changes, VSVIP won't get updated ? I think that would be a problem.

Reply:
seems to be a typo, it should be "+="
fixed it.